src/context/videoContext.js

```diff
@@ -0,0 +1,48 @@
+import React, { createContext, useEffect, useState } from 'react';
+import useVideoApi from '../hooks/useVideoApi';
+
+export const VideosContext = createContext(null);
+
+const VideoContext = ({ children }) => {
+  const { data, loading, error, fetchVideos } = useVideoApi();
+  const [videoSelected, setVideoSelected] = useState(null);
+  const [queryVideo, setQueryVideo] = useState('react');
+
+  useEffect(() => {
+    getVideos();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [queryVideo]);
```

Comment:
The eslint rule react-hooks/exhaustive-deps exists for one reason, it can alert us for possible bugs or unexpected behaviors on the app.

Please remove this ignore rule comment and try to fix the eslint message.